cmd/pulumi-converter-terraform/main.go

```diff
@@ -51,7 +51,7 @@ func (*tfConverter) ConvertProgram(ctx context.Context,
 	src := afero.NewBasePathFs(fs, req.SourceDirectory)
 	dst := afero.NewBasePathFs(fs, req.TargetDirectory)
 
-	diags := tfconvert.TranslateModule(src, dst, providerInfoSource)
+	diags := tfconvert.TranslateModule(src, "/", dst, providerInfoSource)
```

Comment:
```suggestion
	diags := tfconvert.TranslateModule(fs, req.SourceDirectory, dst, providerInfoSource)
```
Give TranslateModule full access to the file system to allow translation of modules that are using relative paths above the root module.